ui/components/NodeText.tsx

```diff
@@ -18,23 +18,50 @@ const Content: FC<Props> = ({ attributes }) => {
       // This text node contains lookup secrets. Let's make them a bit more beautiful!
       // eslint-disable-next-line no-case-declarations
       const secrets = (attributes.text.context as ContextSecrets).secrets.map(
-        (text, k) => (
-          <div
-            key={k}
-            data-testid={`node/text/${attributes.id}/lookup_secret`}
-            className="col-xs-3"
-          >
-            {/* Used lookup_secret has ID 1050014 */}
-            <code>{text.id === 1050014 ? "Used" : text.text}</code>
-          </div>
-        ),
+        (text) => {
+          return text.id === 1050014 ? "Used" : text.text;
+        },
       );
+
       return (
         <div
           className="container-fluid"
           data-testid={`node/text/${attributes.id}/text`}
         >
-          <div className="row">{secrets}</div>
+          <div className="row">
+            <List items={secrets} divided />
+            <div className="u-no-print">
+              <Button
+                type="button"
+                onClick={() => {
+                  const text = secrets.join("\n");
+                  const element = document.createElement("a");
+                  const file = new Blob([text], {
+                    type: "text/plain",
+                  });
+                  element.href = URL.createObjectURL(file);
+                  element.download = "backup-codes.txt";
+                  document.body.appendChild(element);
+                  element.click();
+                  document.body.removeChild(element);
+                }}
```

Comment:
suggestion: I have to ask why inlining this whole function here? Why not relying on `useCallback` ?
Same applies at line 53